### DISCUSSION TITLE
after last firefox update selected tab isn't colored as active

### DISCUSSION THREAD

**Discussion by trofey** [Aris-t2/CustomCSSforFx#632] [Q&A] [answered]
so there is no differencce between active and not active tab
how to fix it?

**Comment by trofey** [comment]
i am sorry that i created new topic i didnt know that it is better towrite in the correct topic

**Comment by Aris-t2** [comment]
Are you using this projects latest files?
Offer a bit more context like settings and also post screenshots.

**Reply by trofey** [reply, marked as answer]
this was too difficult to make new files work like old one, so i ask in reddit and changed selected=true to selected and it worked
